inc/elasticsearch_packages/namespace.php

```diff
@@ -0,0 +1,432 @@
+<?php
+/**
+ * AES Package File Integration.
+ *
+ * @package altis/cloud
+ */
+
+namespace Altis\Cloud\Elasticsearch_Packages;
+
+use Altis;
+use Aws\ElasticsearchService\ElasticsearchServiceClient;
+use Exception;
+use WP_Error;
+
+/**
+ * Hook into search module package support.
+ *
+ * @return void
+ */
+function setup() {
+	// Check the current environment and bail if it isn't AWS.
+	if ( ! in_array( Altis\get_environment_architecture(), [ 'ec2', 'ecs' ], true ) ) {
+		return;
+	}
+
+	// Add a short time out cron hook.
+	// phpcs:ignore WordPress.WP.CronInterval.ChangeDetected
+	add_filter( 'cron_schedules', __NAMESPACE__ . '\\cron_schedules' );
+
+	// Hook into search module.
+	add_filter( 'altis.search.packages_dir', __NAMESPACE__ . '\\packages_dir', 9 );
+	add_filter( 'altis.search.create_package_id', __NAMESPACE__ . '\\create_package_id', 10, 4 );
+	add_filter( 'altis.search.get_package_id', __NAMESPACE__ . '\\get_package_id', 10, 2 );
+	add_action( 'altis.search.check_package_status', __NAMESPACE__ . '\\on_check_package_status', 10, 3 );
+	add_action( 'altis.search.delete_package', __NAMESPACE__ . '\\delete_package', 10, 2 );
+	add_action( 'altis.search.deleted_package', __NAMESPACE__ . '\\on_deleted_package', 10, 4 );
+	add_action( 'altis.search.updated_all_index_settings', __NAMESPACE__ . '\\on_update_index_settings' );
+
+	// Remove default packages updated hook so we don't try to update indexes until
+	// packages have been associated with the domain.
+	add_action( 'plugins_loaded', function () {
+		remove_action( 'altis.search.updated_packages', 'Altis\\Enhanced_Search\\Packages\\on_updated_packages', 10, 2 );
+	}, 20 );
+}
+
+/**
+ * Filter the cron schedules list.
+ *
+ * @param array $schedules The list of available cron schedules.
+ * @return array
+ */
+function cron_schedules( array $schedules ) : array {
+	$schedules['1minute'] = [
```

Comment:
I thought we already had a `minutely` registered? We should probably add these to altis-core anyway really.

Reply:
Not sure, I feel like it needs to be here to be completely reliable.

Reply:
If altis/cloud depends on altis/core, that'd be reliable? IMO, our modules should all depend on core

Reply:
There's no `minutely` schedule registered in any of the Altis modules themselves. I'll update this to be called `minutely` anyway so we could add some additional schedules in the CMS or core module in future

Reply:
👍 I guess it's just a common practice we have. Should def add to altis-core in future, I'll file an issue.